packages/web3/configs/procedure-call.eta

```diff
@@ -96,4 +96,4 @@ const describeReturnType = () => {
         <%~ bodyContentKindTmpl ? `type: ${bodyContentKindTmpl},` : '' %>
         <%~ responseFormatTmpl ? `format: ${responseFormatTmpl},` : '' %>
         ...<%~ _.get(requestConfigParam, "name") %>,
-    }).then(convertHttpResponse)<%~ route.namespace ? ',' : '' %>
+    }).then(<%~ responseBodyInfo.success.schema.contentKind === 'TEXT' ? 'convertTextHttpResponse' : 'convertHttpResponse' %>)<%~ route.namespace ? ',' : '' %>
```

Comment:
Maybe we should add `TEXT` as part of `responseContentKind`, e.g. like [this](https://github.com/alephium/alephium-web3/commit/64772d8f8c23ea805990d7db7a0c961c6484eff1). WDTY?

Reply:
That's fine by me! Feel free to add your commit and continue with the review/merge!